script.js

```diff
@@ -22,13 +23,109 @@ let timer;
 let seconds = 60;
 let userName;
 
+const requestUrl = 'https://opentdb.com/api.php?amount=5&type=multiple';
+const preloader = document.querySelector('.preloader');
+
 
 startBtn.addEventListener('click', checkFormValue);
 checkBtn.addEventListener('click', checkResult);
 endBtn.addEventListener('click', switchSlide);
 restartBtn.addEventListener('click', switchSlide);
 
 
+function sendRequest(url) {
+  return new Promise((resolve,reject) => {
+    let request = new XMLHttpRequest();
+
+    request.open('GET', url);
+
+    request.responseType = 'json';
+
+    let error = new Error('Ошибка при получении вопросов');
+
+    request.onload = () => {
+      if (request.status >= 400) {
+        reject(error);
+      } else {
+        resolve(request.response);
+        preloader.classList.add('none');
+      }
+    };
+
+    request.onerror = () => reject(error);
+
+    request.send();
+  });
+}
+
+
+function processRequest() {
+  sendRequest(requestUrl)
+    .then(response => {
+      let result = response.results;
+
+      let requestQuestions = result.map(item => item.question);
```

Comment:
Codacy has a fix for the issue: [Expected parentheses around arrow function argument.](https://app.codacy.com/gh/voronkova-ab/-game-Who-want-to-be-a-millionaire-/pullRequest?prid=7740148)
```suggestion
      let requestQuestions = result.map((item) => item.question);
```